src/ConfigFactory.Avalonia.Demo/Models/DemoConfig.cs

```diff
@@ -78,4 +81,35 @@ public static ObservableCollection<string> GetThings()
             return value is true;
         });
     }
+
+    /// <summary>
+    /// Example implementation of i18n using the .resx Embedded Resource
+    /// </summary>
+    public override string Translate(string input)
+    {
+        string result = input;
+        PropertyInfo? prop = null;
+
+        if (!string.IsNullOrWhiteSpace(input))
+        {
+            // get resource property
+            prop = (typeof(Translations))
+                .GetProperty(input, BindingFlags.Public | BindingFlags.Static | BindingFlags.NonPublic);
+        }
```

Comment:
This statement should probably return since nothing is done afterwards if `prop` is `null`

---

Left it as is for now, just fixed the formatting

```suggestion
        if (!string.IsNullOrWhiteSpace(input)) {
            // get resource property
            prop = (typeof(Translations))
                .GetProperty(input, BindingFlags.Public | BindingFlags.Static | BindingFlags.NonPublic);
        }
```